docs/Zeek-Customization.md

```diff
@@ -69,3 +66,115 @@ runner without altering its location, any changes will take effect on the next p
 You can specify the location of your Zeek runner via the setting in the **Preferences** menu:
 
 ![Zeek Runner Preference](media/Preferences-Zeek-Runner.png)
+
+# Creating Your Customized Zeek
+
+If you're experienced at [installing Zeek](https://docs.zeek.org/en/current/install/install.html), you may have the best luck
+by building and testing your Zeek installation via those standard instructions. This will give you the most flexibility in
+terms of being able to to compile plugins, use the [Zeek Package Manager](https://docs.zeek.org/projects/package-manager/en/stable/)
+to add extensions, or even make changes to the core Zeek C++ code if you choose.
+
+However, situations may arise when this is difficult. Some examples:
+
+1. **Microsoft Windows**. Zeek is not officially supported on Windows, and Brim's
+[fork of Zeek](https://github.com/brimsec/zeek) that includes limited Windows support is unfortunately not easy to compile
+outside of a customized build environment.
+2. **Minimal changes**. If you seek to only make minor Zeek script additions/changes, creating a full standalone Zeek install
+may seem like overkill.
+3. **Lack of expertise**. The Zeek install/build process may seem daunting to new users.
+
+In such situations, you may be able to start from one of the binary Zeek artifacts that's bundled into Brim and add your
+customizations. However, you should be aware of the following limitations:
+
+1. **Only customizations involving Zeek scripts are possible.** Zeek plugins that include C++ code still require a build
+environment.
+2. **Script additions/changes must be performed manually**, as the
+[Zeek Package Manager](https://docs.zeek.org/projects/package-manager/en/stable/) is not available in the Brim Zeek artifacts.
+
+## Example
+
+The following example shows how to create and use such a customized Zeek, in this case to add the
+scripts from the [zeek-log-all-http-headers](https://github.com/sethhall/zeek-log-all-http-headers) package. PowerShell on
+Windows is used here, but as the steps involve only unpacking and editing files, the equivalent steps on macOS and Linux
+should be intuitive.
+
+1. **Create an empty folder** in which to create the custom Zeek
+
+```
+PS C:\> mkdir \temp\zeek
+PS C:\> cd \temp\zeek
+```
+
+2. **Download and unpack a current Brim Zeek artifact** for the platform from the
+[releases](https://github.com/brimsec/zeek/releases) page.
+
+```
+PS C:\temp\zeek> Invoke-WebRequest -Uri "https://github.com/brimsec/zeek/releases/download/v3.2.0-dev-brim6/zeek-v3.2.0-dev-brim6.windows-amd64.zip" -OutFile zeek.zip
+PS C:\temp\zeek> Expand-Archive zeek.zip
+```
+
+3. **Add/modify the Zeek scripts as necessary.** In this case, we're going to be enabling the Zeek scripts from the
+[zeek-log-all-http-headers](https://github.com/sethhall/zeek-log-all-http-headers) package. Since the scripts are in a GitHub
```

Comment:
```suggestion
3. **Add/modify the Zeek scripts as necessary.** In this case, we're going to add the Zeek scripts from the
[zeek-log-all-http-headers](https://github.com/sethhall/zeek-log-all-http-headers) package. Since the scripts are in a GitHub
```